net/mocks/client_mock.go

```diff
@@ -0,0 +1,295 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
If possible, do not include the auto-generated code in the repo. Instead there should be something which generates this code rather than having the code itself.